galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -19,7 +19,8 @@ def post(settings: Dynaconf) -> Dict[str, Any]:
     data.update(configure_logging(settings))
     data.update(configure_keycloak(settings))
     data.update(configure_cors(settings))
-    data.update(configure_feature_flags(settings))
+    data.update(configure_pulp_ansible(settings))
+    data.update(configure_pulp_ansible(settings))
```

Comment:
You were not supposed to remove `data.update(configure_feature_flags(settings))` from here, this call to configure pulp_ansible is duplicated.